test/test_torch.py

```diff
@@ -5042,6 +5042,130 @@ def test_storage_all_devices(self, devices):
             t = torch.tensor((), device=device)
             self.assertEqual(t.dtype, t.storage().dtype)
 
+    # Note [lazy_clone_ tests with inductor enabled]
```

Comment:
I was able to split the test up into multiple tests to get them to pass when inductor is enabled. This note explains what went wrong and the strategy I used to fix it